Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -162,7 +162,11 @@ def http_request(method, url_suffix, params=None, data=None, files=None, headers
         :param get_token_flag: If set to True will call get_token()
 
         :type no_json: ``bool``
-        :param no_json: If set to true will not parse the content and will return the raw response object for successful response
+        :param no_json: If set to true will not parse the content and will return the raw response object for successful
+        response
+
+        :type status_code: ``int``
+        :param: status_code: The request codes to accept as OK.
```

Comment:
```suggestion
        :param no_json: If set to true will not parse the content and will return the raw response object for successful
        response
        :type status_code: ``int``
        :param: status_code: The request codes to accept as OK.
```

Reply:
What's the differnece here?

Reply:
Just a space

Reply:
looks ok in the code itself. probably git glich
![image](https://user-images.githubusercontent.com/45915502/101637974-221e7b00-3a36-11eb-8be8-48a1054ec458.png)